modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx

```diff
@@ -227,11 +207,7 @@ function DiagramBuilder({
 				connectionMode={ConnectionMode.Loose}
 				edgeTypes={EDGE_TYPES}
 				elements={
-					!isLoadingObjectFolder
-						? elements.length
-							? elements
-							: emptyNode
-						: []
+					!isLoadingObjectFolder && !!elements.length ? elements : []
```

Comment:
Hey @igor-franca, I'm trying to find a reason to make this entire condition, in my tests here it seems that nothing changes if we remove this entire condition, couldn't we just use elements?

```tsx
elements={elements}
```

Reply:
Make sence to me! Thansk